### ISSUE TITLE
看倒你dc发的

### ISSUE THREAD

**Issue by Teabbk** [horw/discord-driven-action#1]
你打算增加什么 或做什么

**Comment by horw** [comment]
I already have a script that helps you open a program and click in a specific order when you receive a message. 
Firstly, I want to add it to this repository. If there are any suggestions, then we can consider doing something else.

**Comment by Teabbk** [comment]
What language does your script use? Python or something?
If I understand correctly, the script you mentioned controls the bot, right?

**Comment by horw** [comment]
Yes, the current script controls the UI for some applications. It was written in Python. 
Perhaps it would be better to make it more universal by using some OpenAI tools.

**Comment by Teabbk** [comment]
Even if you empower it more, it cannot change its narrow applicability.
Have you considered training AI models？
